documentation/modules/auxiliary/scanner/rservices/rsh_login.md

```diff
@@ -0,0 +1,55 @@
+## Vulnerable Application
+
+The R Services (rexecd, rlogind, and rshd) are a suite of unencrypted remote command/login services developed in the 1980s.
+These services are all but unused in modern computing, as they have been replace by telnet and ssh.
+
+`rsh` relies on host names as a security mechanism.  Utilizing `+` can wildcard so any computer can connect.  In the following
```

Comment:
you might note there are exceptions to this rule in proprietary Unix systems. For instance, Kerberos is also supported by the rsh in AIX (though the contents of the session are still unencrypted):

https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.cmds4/rsh.htm

Useful info from a pentester PoV, where you might get access to an AIX LPAR and pivot around.